Makefile.cfg

```diff
@@ -54,16 +71,25 @@ SERIALDEVICE	= /dev/ttyS0
 # port each time.
 #SERIALDEVICE	= /dev/cu.usbserial-5B24
 
-# cygwin & MinGW
-#SERIALDEVICE	= COM1
+# cygwin
+#SERIALDEVICE = COM1
+
+ifdef MINGW32
+	SERIALDEVICE = COM1
+endif
 
 # dir to install dc-tool in
 TOOLINSTALLDIR	= /opt/toolchains/dc/bin
 
 # uncomment this for cygwin
 #EXECUTABLEEXTENSION = .exe
 
+ifdef MINGW32
+	EXECUTABLEEXTENSION = .exe
+endif
 
 # default serial speed - YOU DO NOT NEED TO CHANGE THIS
 SERIALSPEED	= 57600
 
+# Define this if you want a standalone, no dependency binary with MinGW
+STANDALONE_BINARY = 1
```

Comment:
Also, as this is right now, it appears that it will cause all platforms to do a static link by default... Which I don't think is the right thing to do either.

Reply:
I commented out the STANDALONE_BINARY = 1 line